test/spot-checker.js

```diff
@@ -19,12 +17,30 @@ test('fetchCAR - http', async () => {
     protocol: 'http',
     address: '/dns/frisbii.fly.dev/tcp/443/https',
     cid: KNOWN_CID,
-    stats
+    stats,
+    dagScope: 'all'
   })
-  assertEquals(stats.statusCode, 200, 'stats.statusCode')
-  assertEquals(stats.timeout, false, 'stats.timeout')
-  assertEquals(stats.byteLength, 103, 'stats.byteLength')
-  assertEquals(requests, [`ipfs://${KNOWN_CID}?dag-scope=all&protocols=http&providers=%2Fdns%2Ffrisbii.fly.dev%2Ftcp%2F443%2Fhttps`])
+
+  assertEquals(stats.fullRetrieval.statusCode, 200, 'stats.statusCode')
+  assertEquals(stats.fullRetrieval.timeout, false, 'stats.timeout')
+  assertEquals(stats.fullRetrieval.byteLength, 103, 'stats.byteLength')
+
+  await spark.fetchCAR({
+    protocol: 'http',
+    address: '/dns/frisbii.fly.dev/tcp/443/https',
+    cid: KNOWN_CID,
+    stats,
+    dagScope: 'block'
+  })
+
+  assertEquals(stats.blockRetrieval.statusCode, 200, 'stats.statusCode')
+  assertEquals(stats.blockRetrieval.timeout, false, 'stats.timeout')
+  assertEquals(stats.blockRetrieval.byteLength, 103, 'stats.byteLength')
```

Comment:
Ditto.

Reply:
Thanks for feedback; Issue has been addressed in https://github.com/CheckerNetwork/spark-spot-check/pull/6/commits/b5068e3e28f1e841701b98921cf48ef9d2a46b5a